### ISSUE TITLE
HMLR Accessibility Statement Update

### ISSUE THREAD

**Issue by tomguilbert** [epimorphics/hmlr-linked-data#106]
Original request from Eileen Morrison at LR:

> It’s been flagged up to me that the accessibility statement relating to open data needs to be reviewed.
> [Open Data Applications](https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Flandregistry.data.gov.uk%2Faccessibility%3Flang%3Den&data=05%7C01%7CEileen.Morrison%40landregistry.gov.uk%7C01fa53e93683451b5ce708daa60a8516%7C236309a8284b4e57a7b3fc767736f69d%7C0%7C0%7C638004861356669226%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=ymf3xSr52VEJknX6b8tWDPJBTFJM7pK4YCnyPPmtDdA%3D&reserved=0)
> 
> This is overdue for review but fully compliant, last reviewed 22/09/2020
>I believe that Dave liaised with you previously for you to review the accessibility statement?
>I’ve been asked :-
>
>Can you please review the data, make any necessary amendments are made that the last updated date is changed to reflect this review, giving a clear record of compliance to these regulations.

Since there have not been changes since last time this happened, and there is not currently a desire to look for improvements, then we've agreed to conduct a shorter check of the statement for factual changes followed by an update to the date. We would only be able to update the final date on the page, the following would need to remain the same (as we are not going to conduct a full review/test):

> This website has been tested on 21 Sept 2020. Tests were carried out by an external vendor, using a combination of automated testing tools and manually executed test scripts.

HMLR will provide Welsh copy once we provide the wording to translate

**Comment by jonrandahl** [comment]
## Currently the statement reads:
> We know that the SPARQL Qonsole on this website is not fully accessible. In particular, we note the following issues:
> - Background and foreground colours do not have sufficient contrast ratio
>    - This fails WCAG 1.4.3 Contrast (Level AA)
>    - Compatibility work to be completed by end of calendar year 2020
> - Some input fields do not have associated labels in HTML
>    - This fails WCAG 3.3.2 Labels or Instructions (Level A)
>    - Compatibility work to be completed by end of calendar year 2020
> - Lists do not contain only `li` elements and script supporting elements (`script` and `template`)
>    - This fails WCAG 4.1.2 Name, Role, Value (Level A)
>    - Compatibility work to be completed by end of calendar year 2020
> - User cannot progress past main text area using tab key alone
>    - This fails WCAG 2.1.2 No Keyboard Trap
>    - Compatibility work to be completed by end of calendar year 2020
We are working on an update to the SPARQL Qonsole to resolve these issues and make the SPARQL Qonsole WCAG 2.1 compliant. We expect this update to be completed by the end of December 2020.

---

### These issues have been verified to still exist[^1]

![CleanShot 2022-10-07 at 13 35 57](https://user-images.githubusercontent.com/39102/194555351-dc9e704d-af31-4575-8f73-44ed8e71897e.png)

---

### Suggested content revision:

> This statement was reviewed on 07 October 2022.

Alongside a minor typo in the 3rd bullet point supporting elements statement as resolved above

[^1]: colour coded containers describe similar but not exactly the same issues

**Comment by tomguilbert** [comment]
Welsh copy for the sentences in bold above:

Rhyngwyneb technegol sy’n cefnogi datblygiad ymholiad gweledol yw’r SPARQL Qonsole. Rydym yn gweithio ar ddiweddariad i Qonsole SPARQL i wella hygyrchedd yn gyffredinol. Disgwyliwn i’r diweddariad hwn gael ei gwblhau erbyn diwedd Mawrth 2023.

 

Paratowyd y datganiad hwn ar 7 Hydref 2022.

**Comment by jonrandahl** [comment]
@tomguilbert the Welsh translations have been added to the [106 ticket PR](https://github.com/epimorphics/lr-landing/pull/90)

Will update when merged in and ready for review.

**Comment by jonrandahl** [comment]
@tomguilbert closing this as has been resolved and released: https://landregistry.data.gov.uk/accessibility?lang=en
